spec/player_one_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'player_one'
+
+RSpec.describe TicTacToe::FirstPlayer do
+    context "when testing a players move on the board" do
+        it "should return an x" do
+            move = TicTacToe::PlayerOne.new
+            expect(move.choice).to eq("x")
```

Comment:
Remember that `attr_reader :choice` is Ruby short-hand for 
```
def get_choice
  return @choice
end
alias get_choice, choice
```
It's not so much an attribute as it is a method that returns an attribute's current state
When you instantiate your object, `@choice` is nil unless you assign it and you're only assigning it when you call `make_a_move`
And you're not calling 'move.make_a_move' in your test. You're calling `move.choice`

Attr reader is allowing you to reach into the guts of your class and extract what you need instead of asking the only method `make_a_move` that you should really care about.

btw, notice that `move.make_a_move` doesn't make much sense? How would you rename the `move` variable to be something more like what a `PlayerOne.new` is?

This is marvelous work, you are indeed so close!